### ISSUE TITLE
Add button to remove packages from update if they cause transaction errors

### ISSUE THREAD

**Issue by droidmonkey** [timlau/yumex-dnf#123]
When a package causes a transaction error, there should be a button that removes them from the current update batch and continue with the update. See attached screenshot for when this would apply.

![transaction_error](https://cloud.githubusercontent.com/assets/2809491/15913392/b76b9ddc-2da7-11e6-8959-cf1267f533f6.png)

**Comment by timlau** [comment]
This is hard to handle dependecy errors in the repos in a automatic way.
The message you are seeing is just a error message text from dnf, it is hard to parse such a text in a more complex case.
The user must read the message and select the packages with problems in the queue and right click and select **Remove Selected** and re-run the transaction

But will look it if we can make it better in some way

**Comment by timlau** [comment]
Think there is another issue here, yumex show these packages as updates, they are obsoletes

$ sudo dnf update

```
Installing:
 python2-requests                       noarch                       2.10.0-2.fc23                       updates                       107 k
     replacing  python-requests.noarch 2.10.0-1.fc23
 python2-urllib3                        noarch                       1.15.1-3.fc23                       updates                       128 k
     replacing  python-urllib3.noarch 1.15.1-1.fc23
```

So maybe I dont handle them correctly, I will look into that

**Comment by droidmonkey** [comment]
Yes I have noticed that obsoletes tend to "show up" as updates in yumex-dnf but running normal dnf update does not cause this to happen.
